api/tests/Behat/bootstrap/v2/Common/AlertsTrait.php

```diff
@@ -8,16 +8,8 @@ trait AlertsTrait
 {
     public function assertOnAlertMessage(string $alertMessage)
     {
-        $alertDiv = $this->getSession()->getPage()->find('css', 'div.opg-alert--info');
-        if (is_null($alertDiv)) {
-            // fall back to error div
-            $alertDiv = $this->getSession()->getPage()->find('css', 'div.govuk-error-summary');
-            if (is_null($alertDiv)) {
-                $this->throwContextualException(
-                    'A div with the class opg-alert--info was not found. This suggests the page is not what was expected or a condition to display an alert has not been met'
-                );
-            }
-        }
+        $xpath = '//div[contains(@class, \'opg-alert__message\')]|//div[contains(@class, \'opg-alert--info\')]|//div[contains(@class, \'govuk-error-summary\')]';
```

Comment:
😍

Reply:
Although one suggestion - you can lose the `\` if you use `"` instead

Reply:
Yeah I did have it like that and for some reason I preferred the slashes but acually slashes where we don't need them and probably bad.. I dunno. I can change that :)

Reply:
Usually I'm not fussed but when it's stuff like regex or xpath and a slash could actually mean something I tend to try to avoid where possible.